src/ansible/roles/packages/tasks/Debian.yml

```diff
@@ -87,16 +87,27 @@
 
 - name: Install packages for Keycloak base image
   block:
+  - name: Install ca-certificates-java first to avoid dep issues
+    apt:
+      state: present
+      update_cache: yes
+      name:
+      - ca-certificates-java
   - name: Install Keycloak dependencies
     apt:
       state: present
       update_cache: yes
       name:
-      - openjdk-17-jre-headless
       - openssl
       - unzip
       - ca-certificates
-      - ca-certificates-java
+      - openjdk-17-jre-headless
+  - name: Remove openjdk-11 workaround to ca-certificates-java dep issue
+    apt:
+      state: absent
+      update_cache: yes
+      name:
+      - openjdk-11-jre-headless
```

Comment:
Since you install ca-certificates-jave first, Is this still needed or is it a leftover from debugging?

Reply:
I think I was just being overly cautious and removing it in case it caused issues with Keycloak with both 11 and 17 installed.  I removed the code though and the pipelines passed so I'm dropping it.